### ISSUE TITLE
Paren's spacing problem

### ISSUE THREAD

**Issue by Dry8r3aD** [uncrustify/uncrustify#1703]
I actually reported similar issue in the #1699 , and just found out that it's not only for the trenary operator, but about spacing in the multiple parens.

If it's not a bug, i believe that the output code should be like "Expected code #1 " or "Expected code #2". But weirdly, Uncrustify only add space to the "most-outer-closing-paren", but not adding space to the "most-outer-opening-paren".

If there is any more information needed, please comment!

**Uncrusity information**
Linux to Debug file : [debug_new.txt](https://github.com/uncrustify/uncrustify/files/1870617/debug_new.txt)
Link to Config : [uncrustify.txt](https://github.com/uncrustify/uncrustify/files/1870619/uncrustify.txt)
uncrustify version : Uncrustify-0.65_f

Original code
```C
#define NUM_LPM_TESTS (sizeof(tests) / sizeof(tests[0]))
```

Expected code #1 
```C
#define NUM_LPM_TESTS ( sizeof(tests) / sizeof(tests[0]) )
```

Expected code #2 
```C
#define NUM_LPM_TESTS (sizeof(tests) / sizeof(tests[0]))
```

Output code
```C
#define NUM_LPM_TESTS (sizeof(tests) / sizeof(tests[0]) )
```

**Comment by guy-maurel** [comment]
Look at the first comment of #1700

**Comment by gmaurel** [comment]
Solved with PR #2218
